en/docs/intro.md

```diff
@@ -1,19 +1,13 @@
 # Welcome to the High-Speed Data Transmission Customer Portal
-
-The High-Speed Data Transmission (HDT) Portal is a web-based user interface for the HDT platform where you can add, view, and manage your transports and retrieve detailed reports about the usage.
-
+The High-Speed Data Transmission (HDT) Portal is a web-based user interface for the HDT platform, where you can add, view, and manage your transports and retrieve the usage reports.
 
 ## HDT Features
-
-
-- Web and application data travels the fastest path with our over 1,000 Points of Presence in the global regions.
-- Smart routing keeps the best performance in case of network congestion.
-- Multiple paths are used to transmit and merge data, significantly improving performance.
-- High-efficiency service is delivered with our bandwidth-estimation, rapid-retransmission, and fast-recovery technologies.
-- Data encryption and origin IP masking protect content from known and emerging threats.
-- Transport services can be enabled in seconds.
-
+- Access the closest entry and exit points for source and destination with over 1500 Points of Presence in the global regions
+- Smart routing and multi-paths transfer keep the best performance even in network congestions
```

Comment:
"multi-paths" should be "multi-path" in this case.
I suggest rephrasing "keep the best performance even in network congestions" to "maintain the best performance in congested networks"